cassandra/cqlengine/management.py

```diff
@@ -153,7 +153,7 @@ def _get_index_name_by_column(table, column_name):
     Find the index name for a given table and column.
     """
     protected_name = metadata.protect_name(column_name)
-    possible_index_values = [protected_name, "values(%s)" % protected_name]
+    possible_index_values = [protected_name, "values(%s)" % protected_name, "keys(%s)" % protected_name]
     for index_metadata in table.indexes.values():
         options = dict(index_metadata.index_options)
         if options.get('target') in possible_index_values:
```

Comment:
Why does this need to be changed? Was the old line a bug in the test? Or is this an incompatibility between Scylla and Cassandra? Please explain in the commit message, and if it is incompatibility can you link a core issue?

Reply:
There is a bug in scylla core https://github.com/scylladb/scylladb/issues/22019 when it reports index target to a set column a `keys()`, instead of `values()`.
I have removed this change and marked test instead as xfail with link to the issue